### ISSUE TITLE
[Formatting rules] Add support for checking and fixing copyright date in the Kdoc

### ISSUE THREAD

**Issue by akuleshov7** [cqfn/diKTat#3]
If your code has a copyright and it's actual year differs from the current year - it should be fixed (or warning should be raised).
For example if current year is 2020, this code 
```
/** 
* Copyright (c) Ivanoff Technologies Co., Ltd. 1992-2017. All rights reserved
*/
```
should be changed to this one:
```
/** 
* Copyright (c) Ivanoff Technologies Co., Ltd. 1992-2020. All rights reserved
*/
```
